src/Arcus.WebApi.Correlation/HttpCorrelationInfoAccessor.cs

```diff
@@ -27,7 +27,7 @@ public HttpCorrelationInfoAccessor(IHttpContextAccessor contextAccessor)
         /// </summary>
         public CorrelationInfo GetCorrelationInfo()
         {
-            var correlationInfo = _httpContextAccessor.HttpContext.Features.Get<CorrelationInfo>();
+            var correlationInfo = _httpContextAccessor.HttpContext?.Features.Get<CorrelationInfo>();
```

Comment:
Aren't we adding a potential null pointer here? Shouldn't we check if features is null as well?

Reply:
That's a list and is in practice never `null`, but we could, yeah.